pages/en/developer/matchstick.mdx

```diff
@@ -4,15 +4,68 @@ title: Unit Testing Framework
 
 Matchstick is a unit testing framework, developed by [LimeChain](https://limechain.tech/), that enables subgraph developers to test their mapping logic in a sandboxed environment and deploy their subgraphs with confidence!
 
-Follow the [Matchstick installation guide](https://github.com/LimeChain/matchstick/blob/main/README.md#quick-start-) to install. Now, you can move on to writing your first unit test.
+## Getting Started
+
+### Install dependencies
+In order to use the test helper methods and run the tests, you will need to install the following dependencies:
+
+```sh
+yarn add --dev matchstick-as
+```
+
+❗ Since `graph-node` depends on diesel (and diesel requires a local postgres installation) we highly advise using the commands below as adding it in any other way may cause unexpected errors! If you don't have Postgres installed, you will need to install it:
```

Comment:
The dependency on diesel does not really help here and is just confusing in my opinion. It is sufficient to say that graph-node depends on Postgres.